modules/list.js

```diff
@@ -1,20 +1,94 @@
+import { Todobject } from './todo.js';
 /* eslint-disable import/prefer-default-export */
+
 export class Todolist {
+  constructor() {
+    this.todo = [];
+  }
+
+  addList = (input, ind) => {
+    const newList = new Todobject(input, ind);
+    this.todo.push(newList);
+    localStorage.setItem('todo', JSON.stringify(this.todo));
+    this.createlist();
+  }
+
+  deleteList = (index) => {
+    // eslint-disable-next-line eqeqeq
+    this.todo = this.todo.filter((obj) => obj.index != index);
+    this.todo.forEach((item, index) => {
+      item.index = index + 1;
+    });
+    localStorage.setItem('todo', JSON.stringify(this.todo));
+    this.createlist();
+  }
```

Comment:
- [ ] Everything is good to add is working fine, and edit is also working fine but when I try to delete the item then It is not deleting. I would suggest you please fix this issue and also make sure that after deleting it is updating properly on local storage.

Reply:
@roshan-bajgain Please note that I have rechecked and the delete functionality is working just fine.

Reply:
@Koshao-sys, I tried twice and sometimes It used to remove and sometimes when I click twice it was not working.

Reply:
@Koshao-sys, You can ask for another review. Sorry, It's working fine.